# Author: Ilir Librun
# We expect a runtime error in this sample
# We must get the error at line 7 not 9

a = [ref(a),2,3]

for e in a[1]
        a = NULL
next